File: Algorithms/Recursive_algorithms/find_GCD.c
// Write a recursive function to find the GCD (greatest common divisor) of two numbers

#include <stdio.h>

int gcd(int a, int b) {
    if (b == 0) {
        return a;
    } else {
        return gcd(b, a % b);
    }
}

int main() {
    int number1 = 36;
    int number2 = 48;
    int result = gcd(number1, number2);
    printf("GCD of %d and %d is %d\n", number1, number2, result);
    return 0;
}

File: Algorithms/Recursive_algorithms/find_GCD.py
# Write a recursive function to find the GCD (greatest common divisor) of two numbers

def gcd(a, b):
    if b == 0:
        return a
    else:
        return gcd(b, a % b)

number1 = 36
number2 = 48
result = gcd(number1, number2)
print("GCD of", number1, "and", number2, "is", result)